docs/operator-manual/core.md

```diff
@@ -0,0 +1,95 @@
+# Argo CD Core
+
+## Introduction
+
+Argo CD Core is a different installation that runs Argo CD in headless
+mode. With this installation, you will have a fully functional GitOps
+engine capable of getting the desired state from Git repositories and
+applying it in Kubernetes.
+
+The following groups of features won't be available in this
+installation:
+
+- Argo CD RBAC model
+- Argo CD API
+- OIDC based authentication
+- Argo CD web application
+- Argo CD CLI
+- Multi-tenancy
```

Comment:
You can still use Projects to implement a type of multi-tenancy. That multi-tenancy will simply be enforced by git push permissions (strictly gitops) instead of API RBAC.

Reply:
fair.. I created a new section providing a brief explanation.